themes/theme-gmd/widgets/Liveshopping/components/Discount/index.jsx

```diff
@@ -23,7 +23,7 @@ function LiveshoppingDiscount({ discount, productId }) {
       <Portal name={PRODUCT_ITEM_DISCOUNT_BEFORE} props={props} />
       <Portal name={PRODUCT_ITEM_DISCOUNT} props={props}>
         <DiscountBadge
-          text="liveshopping.discount_badge"
+          text={i18n.text('liveshopping.discount_badge', { 0: discount })}
```

Comment:
please apply the translations props as an array like it's usually done for this weird translation placeholders that don't have named placeholders like`{0}`